tox.ini

```diff
@@ -47,6 +47,7 @@ deps =
     lxml==3.4.0
     Twisted==14.0.2
     boto==2.34.0
+    botocore==1.12.89
```

Comment:
I think we should be dropping jessie support, but if we're keeping it, botocore should be 0.62 (see https://packages.debian.org/en/jessie/python-botocore)

Reply:
Updated tox.ini. Waiting for tests to pass. Thanks.

Reply:
botocore version supported by debian jessie does not contain required methods... I'll just skip this test when in jessie environments.